tests/store/test_find_tag.py

```diff
@@ -0,0 +1,28 @@
+"""Tests for finding tags in store."""
```

Comment:
I think some more test cases could be included, for instance to check what happens if filtering is done with a tag name that is not present in the database.

Reply:
Solved! Implemented it in [test_tag_filters.py](tests/store/test_tag_filters.py) instead testing the filter function that is called by this function :)